modules/cli-configuring-kubeconfig-using-cli.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_cli/configuring-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-accessing-kubeconfig-using-cli_{context}"]
+= Accessing `kubeconfig` using the `oc` CLI
+
+You can use the `oc` CLI to log in to your OpenShift cluster and retrieve a kubeconfig file for accessing the cluster from the command line.
+
+.Prerequisites
+
+* You have access to the {product-title} web console or API server endpoint.
+
+.Procedure
+
+. Log in to your OpenShift cluster by running the following command:
++
+[source,terminal]
+----
+$ oc login <api-server-url> -u <username> -p <password> <1>
+----
++
+<1> Replace `<api-server-url>` with the full API server URL, such as `https://api.my-cluster.example.com:6443`.
```

Comment:
Could you also mention/give an example of `<username>` and `<password>` as well?